### PR TITLE
[WIP] prettier example

### PR DESCRIPTION
**Pull request by jumbosushi**
:warning: **SHOUDN'T BE MERGED. THIS IS JUST TO SEE WHAT FORMATTED CODE LOOKS LIKE** :warning: 

Example of what prettier looks like as reffered to in #621 when you run it on js and css files. This PR is just for discussion purposes

Used `prettier --write --no-semi --print-width=120 --single-quote`

### CODE REVIEW

File: ubyssey/static/src/js/components/Article/index.js [hunk 1]
```diff
@@ -3,9 +3,4 @@ import ArticlePreview from './ArticlePreview.jsx'
 import ArticleHeader from './ArticleHeader.jsx'
 import ArticlesSuggested from './ArticlesSuggested.jsx'
 
-export {
-  Article as Article,
-  ArticleHeader as ArticleHeader,
-  ArticlePreview as ArticlePreview,
-  ArticlesSuggested as ArticlesSuggested
-}
+export { Article, ArticleHeader, ArticlePreview, ArticlesSuggested }
```
Comment (jumbosushi) [inline ubyssey/static/src/js/components/Article/index.js:6]: Notable change in JS :thinking:

File: ubyssey/static/src/styles/components/_nationals.scss [hunk 2]
```diff
@@ -17,25 +17,28 @@ $mapHeight: 675px;
...
 .c-n-container {
-  position:relative;
+  position: relative;
   display: flex;
   justify-content: center;
-  h1,h2,h3,h4 {
+  h1,
+  h2,
+  h3,
+  h4 {
```
Comment (jumbosushi) [inline ubyssey/static/src/styles/components/_nationals.scss:41]: Notable change in CSS

File: ubyssey/static/src/styles/modules/_font_faces.scss [hunk 3]
```diff
@@ -1,87 +1,83 @@
...
-	font-weight: 800;
-	font-style: italic;
-	font-stretch: normal;
-	unicode-range: U+0020-25CA;
+  font-family: 'ub-minion-pro';
+  src: url('../fonts/MinionPro-BoldIt_gdi.eot');
+  src: url('../fonts/MinionPro-BoldIt_gdi.eot?#iefix') format('embedded-opentype'),
+    url('../fonts/MinionPro-BoldIt_gdi.woff') format('woff'),
+    url('../fonts/MinionPro-BoldIt_gdi.ttf') format('truetype'),
+    url('../fonts/MinionPro-BoldIt_gdi.svg#MinionPro-BoldIt') format('svg');
```
Comment (jumbosushi) [inline ubyssey/static/src/styles/modules/_font_faces.scss:19]: :100:

File: ubyssey/static/src/styles/modules/article/_gallery.scss [hunk 4]
```diff
@@ -13,10 +12,10 @@ div.slideshow {
...
 
   -webkit-animation: fadein 0.25s; /* Safari, Chrome and Opera > 12.1 */
-   -moz-animation: fadein 0.25s; /* Firefox < 16 */
-    -ms-animation: fadein 0.25s; /* Internet Explorer */
-     -o-animation: fadein 0.25s; /* Opera < 12.1 */
-        animation: fadein 0.25s;
+  -moz-animation: fadein 0.25s; /* Firefox < 16 */
+  -ms-animation: fadein 0.25s; /* Internet Explorer */
+  -o-animation: fadein 0.25s; /* Opera < 12.1 */
+  animation: fadein 0.25s;
```
Comment (jumbosushi) [inline ubyssey/static/src/styles/modules/article/_gallery.scss:18]: :unamused:

**Comment by Rowansdabomb** [comment]
Is there a reason to use prettier over eslint?

**Comment by Rowansdabomb** [comment]
Closing due to inactivity

**Pull request closed without merging**